build/sitemap.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs')
+var path = require('path')
+var async = require('async')
+var packageJson = require('../package.json')
+var inputDir = path.normalize(path.join(__dirname, '..', 'pages'))
+var outputDir = path.normalize(path.join(__dirname, '..', 'dist'))
+
+async.waterfall([
+  function read (done) {
+    fs.readdir(inputDir, done)
+  },
+  function filter (contents, done) {
+    var pages = contents.filter(function (item) {
+      var pathToFile = path.join(inputDir, item)
+      return fs.statSync(pathToFile).isDirectory()
+    }).filter(function (item) {
+      return item !== 'home'
+    })
+    done(null, pages)
+  },
+  function createUrls (pages, done) {
+    var routes = pages.map(function (page) { return 'https://' + packageJson.config.deploy.prod + '/' + page })
```

Comment:
Might be an idea to check this value exists before starting the waterfall and throw an error if not.

I also think our build config should all be in a "tune" property in the `package.json`. Currently, we're scattered in `config.deploy.prod`, `subPages` etc. for another PR perhaps